### ISSUE TITLE
Is the network overfitting?

### ISSUE THREAD

**Issue by John1231983** [junyanz/pytorch-CycleGAN-and-pix2pix#772]
I am using cyclegan to train my own dataset. It has more than 20.000 images.  After training about 1000 epochs.  The G (left) and D loss (right) shown below

![Untitled](https://user-images.githubusercontent.com/24875971/65297439-d718ff00-db35-11e9-9cd9-d68b4fb22220.png)

While the D loss gradually reduces by epoch, the G loss looks increasing or even saturated. What is happen? Which option should I do to solve it?

1. Make G network stronger

2. Make D network weaker. 

3. Use dropout at center block of unet (I used but the problem does not solve)

**Comment by junyanz** [comment]
It is fine as long as D doesn't go to 0. It's normal for  G and D to oscillate.

**Comment by John1231983** [comment]
How about G loss graph. It looks unstable

**Comment by junyanz** [comment]
It looks fine to me. As we are using LSGAN loss, the G and D losses are supposed to oscillate.
